_layouts/default.html

```diff
@@ -61,9 +61,6 @@
         <paper-item label="Developer API (javadoc)" icon="open-in-browser">
           <a href="api/latest/" target="_blank"></a>
         </paper-item>
-        <paper-item label="Releases &amp; Versioning">
```

Comment:
This seemed to get clobbered since it was only committed internally. Not sure why it's only now being clobbered, maybe something to do with an equivalence we forced manually @cgruber?